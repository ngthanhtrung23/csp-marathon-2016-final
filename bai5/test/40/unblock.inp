122334
1.5664
0057..
...788
999RZ.
...RZ.
