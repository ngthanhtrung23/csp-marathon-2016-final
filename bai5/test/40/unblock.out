32
5 D 1
6 L 2
7 U 1
8 L 1
4 D 4
3 R 1
8 R 1
7 D 1
6 R 3
5 U 1
7 U 2
8 L 4
4 U 2
R U 2
Z U 2
9 R 3
5 D 3
0 R 1
8 R 1
1 D 4
0 L 1
2 L 1
8 L 1
5 U 4
0 R 1
8 R 1
1 U 2
9 L 3
4 D 1
R D 1
Z D 1
0 R 3

